cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -207,6 +207,8 @@ func TestGetAPIServerCommand(t *testing.T) {
 				"--enable-admission-plugins=NodeRestriction",
 				"--service-cluster-ip-range=bar",
 				"--service-account-key-file=" + testCertsDir + "/sa.pub",
+				"--service-account-signing-key-file=" + testCertsDir + "/sa.key",
+				"--service-account-issuer=https://kubernetes.default.svc.",
```

Comment:
possibly these unit tests now need the ClusterName appended here?
[1]

Reply:
i dropped cluster name and let user to pass the issuer.